### DISCUSSION TITLE
Compatibility with Python 3.10 and 3.11

### DISCUSSION THREAD

**Discussion by awebster04** [pallets/click#2460] [Q&A] [answered]
When will Click library be compatible with Python 3.10 and 3.11? Is there a future plan to release Click in parallel to Python version and sub version updates?

**Comment by davidism** [comment, marked as answer]
It is compatible. No new release was required. You could have checked this by installing it on that version and observing that it works.

**Reply by awebster04** [reply]
I tried it with both 3.10 and 3.11 versions and it could not find the click module whenever I tried to run my script. When I researched online, I was told that 3.9 was the last compatible version. I installed 3.9 and repointed python and it worked without any additional changes.

**Reply by davidism** [reply]
Not finding the module means you didn't install it correctly. That's an issue with your setup, not with Click's compatibility. Whatever told you 3.9 was the latest supported version was wrong.
